deploy/clowdapp.yaml

```diff
@@ -85,6 +85,18 @@ objects:
               value: ${INSIGHTS_CONTENT_SERVICE_POLL_TIME}
             - name: INSIGHTS_RESULTS_SMART_PROXY__SERVICES__UPGRADE_RISKS_PREDICTION
               value: ${INSIGHTS_DATA_ENGINEERING_URL}
+            - name: INSIGHTS_RESULTS_SMART_PROXY__REDIS__ENDPOINT
+              value: ${INSIGHTS_REDIS_URL}
+            - name: INSIGHTS_RESULTS_SMART_PROXY__REDIS__DATABASE
+              value: ${INSIGHTS_REDIS_DATABASE}
+            - name: INSIGHTS_RESULTS_SMART_PROXY__REDIS__PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: cache-writer-redis-credentials
+                  key: password
+                  optional: true
```

Comment:
I'd drop the `optional`. We should have the secrets in ephemeral, stage and prod. I think it would be easier for debuging to have this as mandatory. This way, if we delete a secret, we will clearly notice with a "secret not found". If we have the optional maybe it will pass and we won't notice

Reply:
done